src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1493,71 +1445,54 @@ namespace HVACUnitaryBypassVAV {
         }
 
         if (state.dataHVACUnitaryBypassVAV->MyPlantScanFlag(CBVAVNum) && allocated(state.dataPlnt->PlantLoop)) {
-            if ((CBVAV(CBVAVNum).HeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) ||
-                (CBVAV(CBVAVNum).HeatCoilType_Num == DataHVACGlobals::Coil_HeatingSteam)) {
-                if (CBVAV(CBVAVNum).HeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+            if ((cBVAV.HeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) || (cBVAV.HeatCoilType_Num == DataHVACGlobals::Coil_HeatingSteam)) {
```

Comment:
`HeatCoilType`

Reply:
These may take a little time to unwind. No need for the string when an enum gives the same information.

        std::string HeatCoilType;                                            // Type of heating coil
        int HeatCoilType_Num = 0;                                            // Numeric equivalent for DX heating coil type